tools/migrate.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Please ensure the variales in migration vars have been assigned before running this script. 
+
+set -euxo pipefail
+
+# shellcheck source=migration_vars
+source migration_vars
```

Comment:
maybe check here that the variables you assume as set are actually set

Reply:
OK yeah, in testing I never bothered to check for this and assumed `-u` would pick that up but it doesn't work if you're sourcing the file.

Reply:
Running the `source <file>` in a subshell fixes this though. New commit up now